site/app/models/User.php

```diff
@@ -513,6 +521,9 @@ public static function validateUserData($field, $data): bool {
             case 'user_legal_familyname':
                 //Given and family name must be alpha characters, latin chars, white-space, or certain punctuation.
                 return preg_match("~^[a-zA-ZÀ-ÖØ-Ýà-öø-ÿ'`\-\.\(\) ]+$~", $data) === 1;
+            case 'user_pronouns':
+                // if the length is less than 12
+                return strlen($data) <= 12;
```

Comment:
Why is length up to 12 required? Should the data validation also include checking for types of characters used like the regex for the given and family names?